### DISCUSSION TITLE
Video latency adjustments slider

### DISCUSSION THREAD

**Discussion by FastenM** [AndreyPavlenko/Fermata#161] [Ideas]
I know there is already a audio latency adjuster, but is there any way you can also add a video latency slider?

More often then not, the bluetooth audio is not synced and even playing with the audio latency slider it doesn't sync up 100% to the video 

So if i can adjust audio and video latency i can probably habe it synced up to 100% sync.

**Comment by AndreyPavlenko** [comment]
The audio latency adjuster does the same thing, but be aware, that it only works for the VLC engine and does not work for Youtube.

**Reply by FastenM** [reply]
Is there anything you can impliment that will work for youtube as well?

**Comment by AndreyPavlenko** [comment]
Unfortunately, no. Youtube is just a web page and I don't have much control over it.

**Reply by FastenM** [reply]
Ok thanks.

**Reply by GoktugUnd** [reply]
> Unfortunately, no. Youtube is just a web page and I don't have much control over it.

Hi @AndreyPavlenko , thank you for all the effort you're putting on this. Appreciate that...
I have been using my good old Samsung S9 for the purpose. Recently I've had a new company car. I moved the same phone & cable to the new car without changing anything and connected to the car (With the same cable) But now there's a latency in between the video and the sound on Youtube !! I know that the latency adjustment sliders won't correct that. But can't figure out what might be causing this and how to resolve. I've read some articles suggesting to turn Wireless Android Auto off. I did that but it didn't solve the issue for me. Do you have any opinion?

**Reply by AndreyPavlenko** [reply]
It could be caused by less powerful hardware in the new car.

**Reply by GoktugUnd** [reply]
> It could be caused by less powerful hardware in the new car.

I doubt that because it is a new Audi (although surprisingly there's no WirelessAA on such a new car :-) ). In fact, after few more days of use, I noticed it doesn't lag on the first video that I watch but it starts lagging on the subsequent ones. So if it was a hardware issue, it would be the same on the first video as well right? 

Also I am experiencing a very unusual issue. The video thumbnails do not load after I slide down the first few lines of videos  &  shorts. This happens if "All videos" from the top menu is selected (Which is the default choice anyway). But if I select latest uploads or the Music, I don't have the same problem. The loading circle turns but thumbnails never appear to choose the video to play.

**Reply by JVKeller** [reply]
> Also I am experiencing a very unusual issue. The video thumbnails do not load after I slide down the first few lines of videos & shorts. This happens if "All videos" from the top menu is selected (Which is the default choice anyway). But if I select latest uploads or the Music, I don't have the same problem. The loading circle turns but thumbnails never appear to choose the video to play.

@GoktugUnd I know how you feel, I can't stand when the audo doesn't match up to the video. 

Have you tried clearing the data for the apps in use? or reinstalling them?

**Reply by GoktugUnd** [reply]
@JVKeller The only app that's in use is Fermata Auto. I tried reinstalling. Nothing has changed. Clearing the data? You mean on Fermata in this case? How?
